test/e2e/infra/vagrant/push_flow_aggregator.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+function usage() {
+    echo "Usage: push_flow_aggregator.sh [-fc|--flow-collector] [-h|--help]
+    Push the latest flow-aggregator image to all vagrant nodes and restart the service
+          --flow-collector   Flow collector is the externalFlowCollectorAddr configMap parameter
+                             It show given in format IP:port:proto. Example: 192.168.1.100:4739:udp"
+}
+
+# Process execution flags
+FLOW_COLLECTOR=""
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+-fc|--flow-collector)
+FLOW_COLLECTOR="$2"
+shift 2
+;;
+-h|--help)
+usage
+exit 0
+;;
+*)
+usage
+exit 1
+esac
+done
+
+if [ "$FLOW_COLLECTOR" == "" ]; then
+    echoerr "--flow-collector must be provided in the expected address format"
+    usage
+    exit 1
+fi
+
+: "${NUM_WORKERS:=1}"
+SAVED_IMG=/tmp/flow-aggregator.tar
+IMG_NAME=antrea/flow-aggregator:latest
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+pushd $THIS_DIR
+
+FLOW_AGG_BASE_YML=$THIS_DIR/../../../../build/yamls/flow-aggregator.yml
+
+FLOW_AGG_YML="/tmp/flow-aggregator.yml"
+
+if [[ $FLOW_COLLECTOR != "" ]]; then
+    $THIS_DIR/../../../../hack/generate-manifest-flow-aggregator.sh --mode dev -fc $FLOW_COLLECTOR > "${FLOW_AGG_YML}"
+else
```

Comment:
Will this `else` condition be triggered? Since at Line 32, if `FLOW_COLLECTOR` is empty string, this script would exit.